bin/rdebug-ide

```diff
@@ -60,6 +60,9 @@ EOB
   opts.on("--attach-mode", "Tells that rdebug-ide is working in attach mode") do
     options.attach_mode = true
   end
+  opts.on("--attach-child", "child process for attach") do
```

Comment:
I feel this explanation is not clear. If I understand the intent correctly, this param defines whether current process is somebody's child thus should not use port setting. Maybe `--ignore-port`?